autogluon/utils/tabular/contrib/tabular_nn_pytorch/nn_tab_model.py

```diff
@@ -0,0 +1,314 @@
+        if self.problem_type == REGRESSION and self.y_scaler is not None:
+            Y_train_norm = pd.Series(self.y_scaler.fit_transform(Y_train.values.reshape(-1, 1)).reshape(-1))
+            Y_test_norm = pd.Series(self.y_scaler.transform(Y_test.values.reshape(-1, 1)).reshape(-1)) if Y_test is not None else None
+            logger.log(15, f'Training with scaled targets: {self.y_scaler} - !!! NN training metric will be different from the final results !!!')
+        else:
+            Y_train_norm = Y_train
+            Y_test_norm = Y_test
+        try:
+            X_train_stats = X_train.describe(include='all').T.reset_index()
+            cat_cols_to_drop = X_train_stats[(X_train_stats['unique'] > self.max_unique_categorical_values) | (X_train_stats['unique'].isna())]['index'].values
+        except:
+            cat_cols_to_drop = []
+        cat_cols_to_keep = [col for col in X_train.columns.values if (col not in cat_cols_to_drop)]
+        cat_cols_to_use = [col for col in self.cat_names if col in cat_cols_to_keep]
+        logger.log(15, f'Using {len(cat_cols_to_use)}/{len(self.cat_names)} categorical features')
+        self.cat_names = cat_cols_to_use
+        self.cat_names = [feature for feature in self.cat_names if feature in list(X_train.columns)]
+        self.cont_names = self.__get_feature_type_if_present('float') + self.__get_feature_type_if_present('int') + self.__get_feature_type_if_present(
+            'datetime')  # + self.__get_feature_type_if_present('vectorizers')  # Disabling vectorizers until more performance testing is done
+        self.cont_names = [feature for feature in self.cont_names if feature in list(X_train.columns)]
+        logger.log(15, f'Using {len(self.cont_names)} cont features')
+        X_train = self.preprocess(X_train, fit=True)
+        if X_test is not None:
+            X_test = self.preprocess(X_test)
+        df_train, train_idx, val_idx = self._generate_datasets(X_train, Y_train_norm, X_test, Y_test_norm)
+        label_class = FloatList if self.problem_type == REGRESSION else None
+        data = (TabularList.from_df(df_train, path=self.path, cat_names=self.cat_names, cont_names=self.cont_names, procs=self.procs)
+                .split_by_idxs(train_idx, val_idx)
+                .label_from_df(cols=LABEL, label_cls=label_class)
+                .databunch(bs=self.params['nn.tabular.bs'] if len(X_train) > self.params['nn.tabular.bs'] else 32))
+        return data
+
+    def _generate_datasets(self, X_train, Y_train, X_test, Y_test):
+        df_train = pd.concat([X_train, X_test], ignore_index=True)
+        df_train[LABEL] = pd.concat([Y_train, Y_test], ignore_index=True)
+        train_idx = np.arange(len(X_train))
+        val_idx = np.arange(len(X_test)) + len(X_train)
+
+        return df_train, train_idx, val_idx
+
+    def fit(self, X_train, Y_train, X_test=None, Y_test=None, **kwargs):
+        logger.log(15, f'Fitting Neural Network with parameters {self.params}...')
```

Comment:
Is there a way to pass `verbosity` to the fastai training to control how much output is printed?